examples/3d/anti_aliasing.rs

```diff
@@ -1,36 +1,95 @@
 //! This example compares MSAA (Multi-Sample Anti-aliasing), FXAA (Fast Approximate Anti-aliasing), and TAA (Temporal Anti-aliasing).
+//!
+//! Add the `--screenshot_taa` to save a TAA motion test screenshot.
 
 // This lint usually gives bad advice in the context of Bevy -- hiding complex queries behind
 // type aliases tends to obfuscate code while offering no improvement in code cleanliness.
 #![allow(clippy::type_complexity)]
 
-use std::f32::consts::PI;
+use std::f32::consts::{FRAC_PI_2, PI, TAU};
 
+use bevy::math::uvec2;
 use bevy::{
+    app::AppExit,
+    core::FrameCount,
     core_pipeline::{
         contrast_adaptive_sharpening::ContrastAdaptiveSharpeningSettings,
         experimental::taa::{
             TemporalAntiAliasBundle, TemporalAntiAliasPlugin, TemporalAntiAliasSettings,
         },
         fxaa::{Fxaa, Sensitivity},
     },
+    math::vec3,
     pbr::CascadeShadowConfigBuilder,
     prelude::*,
+    render::view::screenshot::ScreenshotManager,
+    render::{
+        mesh::VertexAttributeValues,
+        render_resource::{AddressMode, FilterMode},
+    },
     render::{
         render_resource::{Extent3d, SamplerDescriptor, TextureDimension, TextureFormat},
         texture::ImageSampler,
     },
+    window::{PrimaryWindow, WindowResized},
 };
 
+use image::imageops::FilterType;
+
+#[derive(Resource)]
+struct ScreenshotTaa;
+
 fn main() {
-    App::new()
-        .insert_resource(Msaa::Off)
+    let mut app = App::new();
+    let mut movement_settings = CameraMovementSettings::default();
+    if std::env::args().nth(1).unwrap_or_default() == "--screenshot_taa" {
```

Comment:
You could use the `argh` crate for CLI stuff if you prefer. It may be useful for continuous integration / regression test automation to be able to specify which anti-aliasing mode is used. The CI functionality also supports making screenshots at a particular frame. So maybe that's the route to take here.

For example, if you build with the `bevy_ci_testing` feature, you can write `(exit_after: Some(1500), screenshot_frames: [1234])` to a `.ron` file, and then run the example with `CI_TESTING_CONFIG=/path/to/my_config.ron` and it will run the example for that many frames, taking screenshots at the specified frame(s).

Reply:
I would like the test (at least in its current state) to use the frame I specified. Should I add exit_after, screenshot_frames to the CLI options, and also like a test flag, or just have the frame I care about the the default? Do these CLI options need to exist for `bevy_ci_testing` to work or does that work regardless of what options I add?

Reply:
I think I ultimately want to have a set of test scenarios. Like moving the camera side to side, moving the helmets around but keeping the camera in place, etc...